specification/EnterpriseSupport/resource-manager/Microsoft.EnterpriseSupport/preview/2023-05-01-preview/EnterpriseSupport.json

```diff
@@ -0,0 +1,755 @@
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ValidateEnterpriseSupportResponseItem"
+              },
+              "x-ms-identifiers": [],
+              "x-typespec-name": "ValidateEnterpriseSupportResponseItem[]"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ValidateEnterpriseSupport": {
+            "$ref": "./examples/ProviderLevelOperations_ValidateEnterpriseSupport.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "EnterpriseSupports": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/EnterpriseSupportsResponseProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "description": "A ESRPProviderHub resource",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "EnterpriseSupportsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EnterpriseSupports"
+          },
+          "x-typespec-name": "EnterpriseSupports[]",
+          "description": "The EnterpriseSupports items on this page"
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items",
+          "x-typespec-name": "TypeSpec.Rest.ResourceLocation"
+        }
+      },
+      "description": "The response of a EnterpriseSupports list operation.",
+      "required": [
+        "value"
+      ]
+    },
+    "EnterpriseSupportsResponseProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning status"
+        },
+        "billingAccountId": {
+          "type": "string",
+          "description": "Billing Account ID"
+        },
+        "billingGroupId": {
+          "type": "string",
+          "description": "Billing Group ID"
+        },
+        "projectId": {
+          "type": "string",
+          "description": "Invoice section for the billing account"
+        },
+        "productId": {
+          "type": "string",
+          "description": "BicCat product ID"
+        },
+        "skuId": {
+          "type": "string",
+          "description": "BigCat SKU ID"
+        },
+        "availabilityId": {
+          "type": "string",
+          "description": "BigCat availability ID"
```

Comment:
We should avoid exposing BigIds externally... Ideally productCode (a UPN) is the only type of identifier we expose right now.

Reply:
we are going to release this week with P/S/A/T and UPN in the data contract. we will come back to revise this in Aug/Sep timeframe to drop PSAT.